css/style.css

```diff
@@ -0,0 +1,61 @@
+body {
+  font-family: sans-serif;
+  border-top: 5px solid black;
+  border-bottom: 5px solid black;
+  padding-top: 50px;
```

Comment:
@andrewhinderaker this could have written with a shortcut as `padding: 50px 50px` where the first value applies to `top & bottom` and the second value applies to `right & left`